ddp-workspace/projects/ddp-mbc/src/assets/i18n/en.json

```diff
@@ -545,6 +545,16 @@
         },
         "EmailEntry": {
             "InvalidEmail": "Please enter a valid email"
+        },
+        "Institutions": {
+            "Fields": {
+                "Name": "Physician Name",
+                "Institution": "Institution",
+                "InstitutionAny": "Institution (if any)",
+                "City": "City",
+                "State": "State"
+            },
+            "PhysicianValidation": "Please fill out all required fields"
```

Comment:
@zyme, which error message should be here?

Reply:
Looks good as is!